kqueen_ui/blueprints/ui/templates/ui/partial/selectform.html

```diff
@@ -43,7 +43,19 @@
   $.validator.addMethod("yamlfile", validateYamlFileField, $.validator.format("Test validation"));
   $.validator.addMethod(
     "cidr", validateFieldIsIpCidr, $.validator.format("Value doesn't match CIDR format."));
+  $.validator.addMethod('uuid', validateFieldIsUuid, 'Entered value is not a valid UUID.');
 
+  // Prevent two clicks after hiding (https://github.com/twbs/bootstrap/issues/16732)
```

Comment:
Prevent necessity to double-click to open if popover was hidden by el.popover('hide')

But probably you don't need this if you decide to not hide it in 5 sec